src/getTableDefinition.ts

```diff
@@ -31,3 +35,44 @@ export const getTableDefinition = async (
   });
   return tableDefinition;
 };
+
+export const getMultiTableDefinition = async (
+  connection: mysql.Connection,
+  tableSchema: string,
+  tableNames: string[],
+): Promise<MultiTableDefinition> => {
+  const multiTableDefinition: MultiTableDefinition = {};
+  const placeholders = tableNames.map(() => '?').join(',');
```

Comment:
placeholders are used when one need to escape user input for safety, in this case we can just put the values in the query, no need to send it in the other param as there is no safety risk

Reply:
Done